lagom-cloud-object-storage-example/api.sh

```diff
@@ -0,0 +1,29 @@
+
+account.get() {
+  http localhost:9000/api/account/$1/balance --verbose
+}
+
+
+account.deposit() {
+
+cat << EOF > /tmp/last.json
+{
+  "amount": "$2"
+}
+EOF
+  
+  http POST localhost:9000/api/account/$1/deposit --verbose < /tmp/last.json
+}
+
+
+account.withdraw() {
+
+cat << EOF > /tmp/last.json
+{
+  "amount": "$2"
+}
+EOF
+  
+  http POST localhost:9000/api/account/$1/withdraw --verbose < /tmp/last.json
+}
+
```

Comment:
Nice touch! Is the intent that this will be used by the people reading this project? I guess those commands are for HTTPie?

Reply:
I plan to change it to use plan old CURL instead so people don't need HTTPie. It was just easier for me to assemble this one for the moment. 

I will document the URLs to use in any case. People on Linux or Mac will be able to use the `api.sh`, others will need to call using a REST client.

Eventually we could export a Postman project with all the calls prepare. Postman is pretty much popular.